### DISCUSSION TITLE
cosmic ray detector

### DISCUSSION THREAD

**Discussion by Pips06800** [OpenGammaProject/Open-Gamma-Detector#64] [Q&A] [answered]
hello, thank you for creating this affordable project. My goal is to create a cosmic ray detector without the traditional coincidence technique because it takes 2 detectors and I came across someone who detects cosmic rays with a PMT and a Bc412 by selecting only the 4Mev energy range at 30Mev (link: https://physicsopenlab.org/2017/06/26/cosmic-ray-detection-with-gamma-spectrometer/ ,  https://physicsopenlab.org/2019/12/21/muon-rate-monitoring/  ) . The problem is that I don't know if an sipm can detect these impulses in this energy range. Could you do the test for me please? because I haven't purchased the components yet. If this works it will be the best value for money cosmic ray detector that I know.

**Comment by Pips06800** [comment]
I forgot to specify that I use Google translate because I don't speak English, so if you see weird sentences, sorry

**Comment by NuclearPhoenixx** [comment]
Hi there! I think a much better choice of a project to detect muons with a plastic scintillator and SiPM would be the CosmicWatch project. They have built a small all-in-one device, which sounds like a perfect fit for you.

Here is the link: http://www.cosmicwatch.lns.mit.edu/

Hope that helps!

**Comment by Pips06800** [comment]
Thank you for your answer, I already knew the cosmic watch project, it has some problems. we must have 2 detectors, it only uses the coincidence technique. and in addition there is no page where there are all the components like a pcbWays page. what I would like is just to use a single detector and select the energy range from 4Mev or reduce the detection threshold and if that doesn't work I will use your project with the coincidence technique.

**Reply by NuclearPhoenixx** [reply]
I'm not entirely sure what you're planning to do unfortunately, I think some is lost in translation. So do you want to have a directional muon detector or just detect muons in general? I'm not sure I understand why you cannot use CosmicWatch, because you don't _have_ to use the coincidence counting, you can just use it as a single detector, just like you would be able to with my detector. If you want to only count muons then, because you're afraid of skewing the results with natural background radiation, you can always use a well-type lead shield just like in the one openphysicslab post.

**Comment by Pips06800** [comment]
sorry for the translation. what I want to do in general is detect as many muons as possible to make a curve, I don't need to detect the position of the muon. the cosmicWatch project does not allow you to make a spectrometer (even if I know that the bc412 does not really allow you to do spectrometry) unlike your detector which will allow me to select the energy range.

**Reply by NuclearPhoenixx** [reply, marked as answer]
Well, CosmicWatch does have the ability to produce a spectrum with the signal amplitudes. I don't know how well that works, because I've never used it, but it is a feature. There is also lots of other interesting metrics that might be useful:

http://www.cosmicwatch.lns.mit.edu/measure

**Comment by Pips06800** [comment]
Thank you for being so active on your project. I'm going to try to use the cosmicWatch. If I succeed I might even build your detector because it's really incredible.
